architects/demo/res_partner.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_partner_10" model="res.partner">
+        <field name="name">John Doe</field>
+        <field name="is_company" eval="True" />
+        <field name="company_name">John DOe</field>
+    </record>
+    <record id="res_partner_11" model="res.partner">
+        <field name="name">Hello SPRL</field>
+    </record>
+    <record id="res_partner_13" model="res.partner">
+        <field name="name">See the possibilities</field>
+        <field name="is_company" eval="True" />
+        <field name="phone">+1 866-908-3442</field>
+        <field name="website">http://seethepossibilities.nsw.gov.au</field>
+        <field name="email">see@possibilities.com</field>
+        <field name="image_1920" type="base64" file="architects/static/src/binary/res_partner/13-image_1920" />
+    </record>
+    <record id="res_partner_14" model="res.partner">
+        <field name="name">Richard Son</field>
+        <field name="is_company" eval="True" />
+        <field name="email">richardson@example.com</field>
+    </record>
+    <record id="res_partner_15" model="res.partner">
+        <field name="name">Maddie Clarkson</field>
+        <field name="is_company" eval="True" />
+        <field name="email">maddie@gmail.com</field>
```

Comment:
We should avoid having real email, we never know :sweat_smile: So replace gmail/hotmail addresses.
```suggestion
        <field name="email">maddie@example.com</field>
```